com.woltlab.wcf/cronjob.xml

```diff
@@ -202,5 +202,15 @@
 			<startdow>*</startdow>
 			<canbedisabled>0</canbedisabled>
 		</cronjob>
+		<cronjob name="com.woltlab.wcf.exceptionMailer">
+			<classname>wcf\system\cronjob\ExceptionMailerCronjob</classname>
+			<description>Sends out notifications about new entries in the error log.</description>
+			<description language="de">Versendet Benachrichtigungen über neue Einträge im Fehlerprotokoll.</description>
```

Comment:
For reasons of consistency, I would use the following wording: `Versendet Benachrichtigungen über neue protokollierte Fehler.`.